http/src/v0/refs.rs

```diff
@@ -135,19 +134,26 @@ async fn refs_paths<T: IpfsTypes>(
     use futures::stream::FuturesOrdered;
     use futures::stream::TryStreamExt;
 
-    // the assumption is that futuresordered will poll the first N items until the first completes,
-    // buffering the others. it might not be 100% parallel but it's probably enough.
-    let mut walks = FuturesOrdered::new();
+    let opts = WalkOptions {
+        follow_dagpb_data: true,
+    };
 
-    for path in paths {
-        walks.push(walk_path(&ipfs, path));
-    }
+    // added braces to spell it out for borrowck that opts does not outlive this fn
```

Comment:
at this point in time `WalkOptions` could just derive `Copy`, but I'm ok with not doing that if we're expecting more advanced options later on

Reply:
yeah don't know whats going to happen for this in the follow ups. I am hesitant to make it pretty while I'd like to find a way to get rid of this if possible.

Reply:
if it's not possible to get rid of it, perhaps turning the resolving to a "Command" alike with builder and so on might be necessary. I think this will be related to how the solution for the problematic `refs` timeout test will pan out..